GlobalConstantsService/src/ParticleDataList.cc

```diff
@@ -0,0 +1,121 @@
+
+#include <fstream>
+#include <iomanip>
+#include <iostream>
+
+#include "cetlib_except/exception.h"
+
+#include "Offline/GlobalConstantsService/inc/ParticleDataList.hh"
+#include "Offline/ConfigTools/inc/ConfigFileLookupPolicy.hh"
+#include "Offline/ConfigTools/inc/SimpleConfig.hh"
+#include "Offline/DataProducts/inc/PDGCode.hh"
+
+
+namespace mu2e {
+
+  // *************************************************************
+
+  ParticleDataList::ParticleDataList( SimpleConfig const& config ) {
+
+    ConfigFileLookupPolicy findConfig;
+    
+    std::string tableFilename = findConfig(
+                    config.getString("particleDataList.filename",
+                   "Offline/GlobalConstantsService/data/ParticleList.txt") );
```

Comment:
I recommend no default on the grounds that if "particleDataList.filename" is intended to be in the fcl file, but is misspelled, I would like it to be an error.

Reply:
thanks.